tests/e2e/adu/service/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "ts-e2e",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "npmlockrefresh": "npm i --package-lock-only",
+    "build": "tsc",
+    "cleanup": "node ./lib/common/e2e_test_cleanup.js --connectionString ${ADU_IOTHUB_CONNECTION_STRING} --e 5",
+    "adu": "mocha ./lib/adu_client/e2e_adu_client_test.js  --connectionString ${ADU_IOTHUB_CONNECTION_STRING} --e ${DEVICE_TEST_EXE} --timeout 120000"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",

```

Comment:
Should this be MIT since most of the rest of our stuff is MIT?

Reply:
do json files allow comments? I thought they didn't

Reply:
Is that a requirement of using the MIT license?

Reply:
Oh oh oh I see now what you pointed to sorry. Yes you're right. When I hear MIT license my brain always goes to the comments on the top of files. I thought you were commenting on the file as a whole my bad.